src/software/world/robot_state.h

```diff
@@ -60,8 +62,15 @@ class RobotState
     AngularVelocity angularVelocity() const;
 
     /**
-     * Defines the equality operator for a RobotState. RobotStates are equal if their
-     * positions, velocities, orientation, and angular velocity are the same
+     * Returns the height of the robot in metres
+     *
+     * @return the height of the robot in metres
+     */
+    double height() const;
```

Comment:
Is this for hybrid fleet? Enemy robots? 
Another point is that this is RobotState, with the assumption that these values can change over time. I don't think that's the case for robots, so it doesn't belong in the RobotStatae class

Reply:
> Is this for hybrid fleet? Enemy robots?

For whatever robot this state is created for. Values may differ for hybrid fleet, and perhaps we want to assume enemy robots are as tall as legally allowed so we can always chip over them. We actually can get robot height data from SSL-Vision/Gamecontroller.

> Another point is that this is RobotState, with the assumption that these values can change over time. I don't think that's the case for robots, so it doesn't belong in the RobotStatae class

I guess I wasn't quite thinking of the state as something where all values necessarily change over time, more just that it's the minimal set of info to represent a robot at an arbitrary point in time. For example if we put the `height` in the Robot class I had a function `canChipOverRobot(Robot)`, then I'd be providing a lot of extra unneeded data to the function (state history, timestamps). Breaking out the `state` classes should help avoid this. At least for that reason I think this is a reasonable place for `height`.

Reply:
I get that we don't want to use the full Robot class to check if we can chip over a robot. What about calculate `canChipOverRobot` using the max height of the robot and not being as precise about the exact height of robots? I'm wondering how big of a difference this makes compared to adding another variable to RobotState? @akhilveeraghanta @garethellis0

Reply:
Yea always using max height is totally reasonable, robots are only every going to vary by a few centimeters which doesn't really change much. Part of the reason I also added it was thinking ahead to avoid more global constants.

If our ultimate goal with DynamicParams is to remove magic global constants, these these values eventually have to come from _somewhere_, so figured adding the height to the state was at least as reasonable as potentially injecting a config (or passing a config along with a state to function like `canChipOverRobot`, although that's actually not too bad I think).

Reply:
I don't think that the max allowable robot height by SSL is a magic global constant. I would accept that sort of constant

Reply:
ok I removed the `height` from robot state and use the constant in the translation.